src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        String color = new ColorSupplier().getRandomColor();
+        int number = new Random().nextInt(101);
```

Comment:
1) You dont want to create Supplier and Random objects each time getRandomBall calls, move to field declaration. 
2) magic number